README.md

```diff
@@ -259,6 +261,12 @@ Busboy methods
 
     * The constructor can throw errors:
 
-        * **Unsupported content type: $type** - The `Content-Type` isn't one Busboy can parse.
+        * **Busboy expected an options-Object.** - Busboy expected an Object as first parameters.
+
+        * **Busboy expected an options-Object with headers-attribute.** - The first parameter is lacking of a headers-attribute.
+
+        * **Limit $limit is not a valid number** - Busboy expected the desired limit to be of type number. Busboy throws this Error to prevent a potential security issue by falling silently back to the Busboy-defaults. Potential source for this Error can be the direct use of environment variables without transforming them to the type number. 
```

Comment:
See utils.js/getLimit
Should we move those checks to the constructor so that this only has overhead on instantiation, instead of during the runtime?

Reply:
Afaik the use case is actually, that busboy is initialized for every request. So it is not like, we initialize busboy once and then we reuse it but we always have to check all the limits on every request. 

wdyt

Reply:
yeah, fair.